package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4715,6 +4726,47 @@ def _matches_unpickling(self, anchor_name, n_atoms, fname, trajname):
         else:
             return False
 
+    def transfer_to_memory(self, frame_interval=1):
+        """Replace the current trajectory reader object with one of type
+        :class:`MDAnalysis.coordinates.memory.MemoryReader` to support in-place
+        editing of coordinates.
+
+        :Arguments:
+             *frame_interval*
+                 Read in every nth frame.
+         """
+
+        from ..coordinates.memory import MemoryReader
+
+        if self.trajectory.format != "array":
+
+            # Try to extract coordinates using Timeseries object
+            # This is significantly faster, but only implemented for certain
+            # trajectory file formats
+            try:
+                coordinates = self.universe.trajectory.timeseries(
+                    self.atoms, format='afc', skip=frame_interval)
+
+            # if the Timeseries extraction fails,
+            # fall back to a slower approach
+            except AttributeError:
+                coordinates = np.zeros(
+                    tuple([self.universe.trajectory.n_frames/frame_interval]) +
+                    self.atoms.coordinates().shape)
+
+                k = 0
+                for i, time_step in enumerate(self.universe.trajectory):
```

Comment:
Yeah, this is what DCD.timeseries() does, so I followed that lead. I would also think it was nicer if it followed the numpy stride convention instead, so I would be happy to change it - but for consistency we should probably change the DCDReader C-implementation as well then.

I just added (f26cd38b240c462c1bd2ff18105bc76ecd45a69d) this test in test_atomgroup.py to verify the behavior:
```python
    def test_frame_interval_convention():
        universe1 = MDAnalysis.Universe(PSF, DCD)
        array1 = universe1.trajectory.timeseries(skip=10)
        universe2 = MDAnalysis.Universe(PSF, DCD, in_memory=True,
                                        in_memory_frame_interval=10)
        array2 = universe2.trajectory.timeseries()
        assert_equal(array1, array2,
                     err_msg="Unexpected differences between arrays.")
```